overlay_gl/overlay.c

```diff
@@ -273,6 +282,17 @@ static void drawOverlay(Context *ctx, unsigned int width, unsigned int height) {
 			return;
 		}
 		ods("Socket connected");
+
+		struct OverlayMsg om;
+		om.omh.uiMagic = OVERLAY_MAGIC_NUMBER;
+		om.omh.uiType = OVERLAY_MSGTYPE_PID;
+		om.omh.iLength = sizeof(struct OverlayMsgPid);
+		om.omp.pid = getpid();
+
+		if (!sendMessage(ctx, &om))
```

Comment:
So we send an overlaymsg for every frame/swap now?
Is this really necessary? What for?